archbuild.in

```diff
@@ -69,7 +69,7 @@ if ${clean_first} || [[ ! -d "${chroots}/${repo}-${arch}" ]]; then
 
 	rm -rf --one-file-system "${chroots}/${repo}-${arch}"
 	(umask 0022; mkdir -p "${chroots}/${repo}-${arch}")
-	setarch "${arch}" mkarchroot \
```

Comment:
please do not remove setarch here but preserve it in a comparable way like `arch-nspawn` does with a `${CARCH:+setarch "$CARCH"}` type of call